divisibleBy35/app.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+//
+// invoke with go run ./app.go 1 100
+//
+
+// import the libraries we need
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+
+// golang is a typed language so we delcare that. 
+// no return expected
+func numsBetweenTwoAndFive(one int, two int) {
```

Comment:
I don't understand this function name... `numsBetweenThreeAndFive` maybe? (Although probably worth noting, I think we all agree we wouldn't want to lock this codebase down into just those divisors.

Reply:
It should be three. I... had leetcode on the brain. Specifically "twosum", and I did it for both python and golang. Caught it on python... so yes, numsBetweenThreeAndFive.